Sitemap.php

```diff
@@ -145,6 +145,13 @@ public function addItem($location, $lastModified = null, $changeFrequency = null
         }
 
         $this->writer->startElement('url');
+
+        if(false === filter_var($location, FILTER_VALIDATE_URL)){
+            throw new \InvalidArgumentException(
+                'The location must be a valid URL' . '. You have specified: ' . $location . '.'
```

Comment:
Seems better (+1)
@samdark what do you think ? personally, I am use single quotes a lot